src/css/main.scss

```diff
@@ -450,6 +455,9 @@ button {
     @include media('lg-up') {
       padding-top: 3.8125rem;
       padding-bottom: 4.625rem;
+      // removing site-padding-x
```

Comment:
removing `.site-padding-x` from both article header and body — matches designs better for 1250px+ screens